FBSimulatorControlTests/Tests/Integration/FBSimulatorDiagnosticsTests.m

```diff
@@ -43,7 +43,9 @@ - (void)testAppCrashLogIsFetched
   }
 
   FBSimulator *simulator = [self assertObtainsBootedSimulator];
-  FBApplicationLaunchConfiguration *appLaunch = [self.tableSearchAppLaunch.injectingShimulator withEnvironmentAdditions:@{@"SHIMULATOR_CRASH_AFTER" : @"1"}];
+  NSString *path = [[NSBundle bundleForClass: self.class] pathForResource:@"libShimulator" ofType:@"dylib"];
```

Comment:
Is the `injectingShimulator` method working without needing to make this change? The Change to the `pbxproj` should be sufficient.